src/ensembl/src/content/app/entity-viewer/gene-view/components/proteins-list/proteins-list-item-info/ProteinsListItemInfo.tsx

```diff
@@ -224,19 +240,17 @@ const ProteinsListItemInfo = (props: Props) => {
             )}
           </div>
           {proteinSummaryStats &&
-            uniprotXref &&
+            proteinXrefs &&
```

Comment:
if `proteinXrefs` is returned by `getProteinXrefs`, it is always truthy. An empty array is still truthy.